rfcs/packaging/package-structure.md

```diff
@@ -0,0 +1,134 @@
+# RFC: Communicating new package structure for `@fluentui/*`
+
+**Note that this RFC focuses solely on the package structure from a consumer perspective, focused primarily on v7/8 and converged packages. It does not deal with folder structure or northstar packages.**
+
+Contributors: @paulgildea, @JustSlone, @ecraig12345, @khmakoto
+
+## Summary
+
+Currently, we have a lot of different component packages, and it's not clear to consumers what the purpose of each one is or whether it's production-ready.
+
+There are a few aspects to the proposed solution:
+
+- Add badges (or similar) to package readmes to clarify GA/preview/dev status
+- "Converged" component packages only contain convergence work
+  - Move old (v8) component implementations from `react-{checkbox,link,slider,tabs,toggle}` back to `react-internal`
+  - Move prototype "converged" components to the package roots and get rid of `/next` folders
+  - Probably also move all component implementations in `react` back to `react-internal`
```

Comment:
Just to confirm, does "all component implementations in `react`" refer to these components?
https://github.com/microsoft/fluentui/tree/master/packages/react/src/components

Reply:
Yes